src/kontrol/__main__.py

```diff
@@ -806,6 +808,13 @@ def _parse_test_version_tuple(value: str) -> tuple[str, int | None]:
         action='store_true',
         help='Flag used by Simbolik to initialise the state of a non test function as if it was a test function.',
     )
+    prove_args.add_argument(
+        '--hevm',
+        dest='hevm',
+        default=False,
+        action='store_true',
+        help='Uses the hevm success predicate',
```

Comment:
Something like "Runs the tests against the Hevm success predicate instead of the default Foundry's success criteria" is ok? Or would you suggest something else?

Reply:
How about "Use `hevm` success predicate instead of `foundry` to determine if a test is passing", or something like that?